conf/scalardb-custom-values.yaml

```diff
@@ -14,42 +14,45 @@ envoy:
     type: LoadBalancer
     annotations:
       service.beta.kubernetes.io/azure-load-balancer-internal: "true"
-      service.beta.kubernetes.io/azure-load-balancer-internal-subnet: "k8s_ingress"
       service.beta.kubernetes.io/aws-load-balancer-internal: "true"
       service.beta.kubernetes.io/aws-load-balancer-type: "nlb"
 
-  resources: {}
+  resources:
+    requests:
+      cpu: 200m
+      memory: 256Mi
+    limits:
+      cpu: 300m
+      memory: 328Mi
 
   tolerations:
     - effect: NoSchedule
-      key: kubernetes.io/app
+      key: scalar-labs.com/dedicated-node
       operator: Equal
-      value: scalardbpool
+      value: scalardb
 
   affinity:
     nodeAffinity:
       requiredDuringSchedulingIgnoredDuringExecution:
         nodeSelectorTerms:
           - matchExpressions:
-              - key: agentpool
+              - key: scalar-labs.com/dedicated-node
                 operator: In
                 values:
-                  - scalardbpool
+                  - scalardb
     podAntiAffinity:
-      preferredDuringSchedulingIgnoredDuringExecution:
-        - podAffinityTerm:
-            labelSelector:
-              matchExpressions:
-                - key: app.kubernetes.io/name
-                  operator: In
-                  values:
-                    - scalardb
-                - key: app.kubernetes.io/app
-                  operator: In
-                  values:
-                    - envoy
-            topologyKey: kubernetes.io/hostname
-          weight: 50
+      requiredDuringSchedulingIgnoredDuringExecution:
```

Comment:
If we use `preferredDuringSchedulingIgnoredDuringExecution`, it is possible to deploy several Scalar pods on the same worker node since it doesn't force the rules (one Scalar pod on the one worker node).
In other words, there is a possibility that the pod status will not be `PENDING` and Cluster Autoscaler doesn't work.

To make Cluster Autoscaler work properly and keep `one Scalar pod on the one worker node`, I used the `requiredDuringSchedulingIgnoredDuringExecution` instead.

The `requiredDuringSchedulingIgnoredDuringExecution` forces this rule.
So, the Cluster Autoscaler detects a `PENDING` pod when the Scalar pods are scaled out, and we can keep `one Scalar pod on the one worker node`.